src/Component/Nav/Nav.scss

```
@@ -0,0 +1,43 @@
+.navBar {
+  .navWrap {
+    display: flex;
+    align-items: center;
+    height: 68px;
+    border-bottom: 1px solid #eee;
+  }
+  .navContainer {
+    width: 1247px;
+    height: 48px;
+    padding-top: 2px;
+    margin: 0 auto;
+  }
+
+  img {
+    width: 152px;
+    height: 40px;
+  }
+
+  .navRightBar {
+    float: right;
+    padding: 9px 0 9px 0;
+
+    button {
+      border: 0;
+      outline: 0;
+      background-color: #fff;
+      size: 13px;
+      color: #db2f8b;
+    }
+
+    .navRightWonBtn {
+      padding-right: 11px;
+    }
+
+    .navLogin {
+      padding: 6px 15px;
+      border: 1px solid #555;
+      border-radius: 26px;
+      color: black;
+    }
+  }
+}
```

```suggestion
.navBar {
  display: flex;
  align-items: center;
  justify-content: space-between;
  height: 68px;
  padding: 0 50px;
  border-bottom: 1px solid #eee;

  img {
    width: 152px;
    height: 40px;
  }

  button {
    border: 0;
    outline: 0;
    background-color: #fff;
    color: #db2f8b;
  }

  .navRightWonBtn {
    padding-right: 11px;
  }

  .navLogin {
    padding: 6px 15px;
    border: 1px solid #555;
    border-radius: 26px;
    color: black;
  }
}
```
이런식으로 더 간단하게 바꿀 수 있겠죠?
그리고 button에 size라는 속성은 없습니다!